src/wp-includes/kses.php

```diff
@@ -2467,10 +2468,11 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 		}
 
 		if ( $found ) {
-			// Allow CSS calc().
-			$css_test_string = preg_replace( '/calc\(((?:\([^()]*\)?|[^()])*)\)/', '', $css_test_string );
+			// Allow some CSS functions.
+			$css_test_string = preg_replace( '/\b(?:calc|min|max|minmax|clamp)\(((?:\([^()]*\)?|[^()])*)\)/', '', $css_test_string );
```

Comment:
Nit: The outermost capturing group is redundant.

```suggestion
			$css_test_string = preg_replace( '/\b(?:calc|min|max|minmax|clamp)\((?:\([^()]*\)?|[^()])*\)/', '', $css_test_string );
```